lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if ('max_team_size' in course_module.teams_configuration and
+            team.users.count() >= course_module.teams_configuration['max_team_size']):
```

Comment:
As a way of simplifying this line, though, you should be able to do something like: `if course.max_team_size and team.users.count() >= course.max_team_size` instead, using `[max_team_size](https://github.com/edx/edx-platform/blob/01065b6ae5f998fe984aeaa6849fe6adf79e844f/common/lib/xmodule/xmodule/course_module.py#L1524)` for convenience.